geode-core/src/test/java/org/apache/geode/management/ConcurrencyTestHelperTest.java

```diff
@@ -0,0 +1,190 @@
+    } catch (InterruptedException e) {
+      throw new IllegalStateException("Interrupted");
+    }
+
+    return Boolean.TRUE;
+  };
+
+  public Callable<Boolean> c3 = () -> {
+    throw new IllegalStateException("Expect this");
+  };
+
+  public Callable<Boolean> c4 = () -> {
+    try {
+      Thread.sleep(25000);
+    } catch (InterruptedException e) {
+      throw new IllegalStateException("Interrupted");
+    }
+
+    return Boolean.TRUE;
+  };
+
+  public Runnable r1 = () -> {
+    r1Status = Boolean.TRUE;
+  };
+
+  public Runnable r2 = () -> {
+    try {
+      Thread.sleep(350000);
+    } catch (InterruptedException e) {
+      throw new IllegalStateException("Interrupted");
+    }
+
+    r2Status = Boolean.TRUE;
+  };
+
+  public Runnable r3 = () -> {
+    r3Status = Boolean.TRUE;
+    throw new IllegalStateException("Expect this");
+  };
+
+  public Runnable r4 = () -> {
+    try {
+      Thread.sleep(25000);
+    } catch (InterruptedException e) {
+      throw new IllegalStateException("Interrupted");
+    }
+
+    r4Status = Boolean.TRUE;
+  };
+
+  @Before
+  public void before() {
+    r1Status = Boolean.FALSE;
+    r2Status = Boolean.FALSE;
+    r3Status = Boolean.FALSE;
+    r4Status = Boolean.FALSE;
+  }
+
+  @Test
+  public void testRunnablesWithDefaultTimeout() throws InterruptedException {
+    List<Runnable> runnables = Arrays.asList(r1, r2, r3, r4);
+    List<Throwable> exceptions = ConcurrencyTestHelper.runRunnables(runnables);
+
+
+    assertThat(exceptions.get(0)).isNull();
+    assertThat(exceptions.get(1)).isInstanceOf(CancellationException.class);
+    assertThat(exceptions.get(2)).isInstanceOf(IllegalStateException.class)
+        .hasMessageContaining("Expect this");
+    assertThat(exceptions.get(3)).isNull();
+
+    assertThat(r1Status).isTrue();
+    assertThat(r2Status).isFalse();
+    assertThat(r3Status).isTrue();
+    assertThat(r4Status).isTrue();
+  }
+
+  @Test
+  public void testRunnablesWithNonDefaultTimeout() throws InterruptedException {
+    List<Runnable> runnables = Arrays.asList(r1, r2, r3, r4);
+    List<Throwable> exceptions = ConcurrencyTestHelper.runRunnables(runnables, 20);
+
+    assertThat(exceptions.get(0)).isNull();
+    assertThat(exceptions.get(1)).isInstanceOf(CancellationException.class);
+    assertThat(exceptions.get(2)).isInstanceOf(IllegalStateException.class)
+        .hasMessageContaining("Expect this");
+    assertThat(exceptions.get(3)).isInstanceOf(CancellationException.class);
+
+    assertThat(r1Status).isTrue();
+    assertThat(r2Status).isFalse();
+    assertThat(r3Status).isTrue();
+    assertThat(r4Status).isFalse();
+  }
+
+  @Test
+  public void testCallablesWithDefaultTimeoutWithNoExceptions() {
+    List<Callable<Boolean>> callables = Arrays.asList(c1, c4);
+    List<Boolean> results = null;
+
+    try {
+      results = ConcurrencyTestHelper.runCallables(callables);
+    } catch (Exception e) {
+      fail("Expected no exceptions to be thrown.");
```

Comment:
The expectations for the Callable tests with no expected exceptions it's not really necessary to do the `runCallables()` in a try/catch. Just add a throws declaration to the test methods and let any exceptions propagate up to the test runner where they will cause a test failure.